docs/app/src/errors.js

```diff
@@ -34,6 +34,10 @@ angular.module('errors', ['ngSanitize'])
 
 
 .directive('errorDisplay', ['$location', 'errorLinkFilter', function ($location, errorLinkFilter) {
+  var encodeAngularBrackets = function (text) {
```

Comment:
I think "angle brackets" might be more "correct" (and certainly less confusing).